packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -19,8 +19,10 @@ import labelStyles from '@adobe/spectrum-css-temp/components/fieldlabel/vars.css
 import {mergeProps} from '@react-aria/utils';
 import React, {RefObject} from 'react';
 import {SpectrumFieldProps} from '@react-types/label';
+import {useFormProps} from '@react-spectrum/form';
 
 function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
+  props = useFormProps(props);
```

Comment:
Will need to update MobileComboBox in such a way that the tray input field isn't affected by the FormProps, don't want this to happen:
![image](https://user-images.githubusercontent.com/9661127/134253878-0b25ab51-31d8-4041-837d-e5f9bb50c302.png)

Reply:
Probably just need to do the following here: https://github.com/adobe/react-spectrum/blob/11596864232ede67642f304e175a678eafcbb176/packages/%40react-spectrum/combobox/src/MobileComboBox.tsx#L441-L443

```
        <TextFieldBase
          labelAlign="start"
          labelPosition="top"
```

Reply:
I've been thinking about this. I don't want to remove useFormProps() from Field. The tray uses a TextFieldBase and I don't recall a way to unset inherited props, which means I need to manually set the labelPosition to top and the labelAlign to start in the trays TextFieldBase.